integration/spark-common/src/main/scala/org/apache/spark/DataSkewRangePartitioner.scala

```diff
@@ -202,7 +202,7 @@ class DataSkewRangePartitioner[K: Ordering : ClassTag, V](
     }
   }
 
-  private val skewPartitions: Int = if (skewCount == 0) {
+  private var skewPartitions: Int = if (skewCount == 0) {
```

Comment:
What does skewPartitions and skewCount mean respectively?

Reply:
shewCount: the count of shew index
skePartitions: the number of skew partitions